pkg/config/json_test.go

```diff
@@ -15,11 +15,11 @@ type jsonTest struct {
 var jsonTests = map[string]jsonTest{
 	"TestLinuxVM": {
 		newVM:        newLinuxVM,
-		expectedJSON: `{"vcpus":3,"memoryBytes":4000000000,"bootloader":{"kind":"linuxBootloader","VmlinuzPath":"/vmlinuz","KernelCmdLine":"/initrd","InitrdPath":"console=hvc0"}}`,
+		expectedJSON: `{"vcpus":3,"memoryBytes":4194304000,"bootloader":{"kind":"linuxBootloader","vmlinuzPath":"/vmlinuz","kernelCmdLine":"/initrd","initrdPath":"console=hvc0"}}`,
```

Comment:
`"kernelCmdLine":"/initrd","initrdPath":"console=hvc0"`

I suddenly realized that this JSON seems to be incorrect, the correct format should be:`"initrdPath":"/initrd","kernelCmdLine": "console=hvc0",`. Perhaps we should fix this issue in another PR?

Reply:
Ah good catch, I've just fixed this, it was only an issue in json_test.go, the arguments to NewLinuxBootloader are not in the expected order.